### DISCUSSION TITLE
Brew wants Xcode 13.2.1, but no such download/update is available (already have CLT)

### DISCUSSION THREAD

**Discussion by benknoble** [Homebrew/discussions#3846] [Everyday usage] [answered]
```
# brew config
HOMEBREW_VERSION: 3.6.7-32-g31758ea
ORIGIN: https://github.com/Homebrew/brew
HEAD: 31758eaa0272a7860816cf584ed91a72a95a1f7f
Last commit: 30 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a9064e28a1ac1ffad78c57fdd49b554d41152d0d
Core tap last commit: 11 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.AcWGLWikRj/org.macosforge.xquartz:0
HOMEBREW_EDITOR: /usr/local/bin/vim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_AUTO_UPDATE: set
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8_1/bin/ruby
CPU: octa-core 64-bit haswell
Clang: 11.0.0 build 1100
Git: 2.36.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.7-x86_64
CLT: 13.2.0.0.1.1638488800
Xcode: 11.3.1
```

I've recently update to Big Sur<sup>(I know, I'm way behind: I was coming from Mojave, so cut me a break)</sup>, and updated Homebrew and packages as well.

Running `brew upgrade` (after successful `brew update`) shows
```
Error: Your Xcode (11.3.1) is too outdated.
Please update to Xcode 13.2.1 (or delete it).
Xcode can be updated from the App Store.
```

I'm wary of deleting it. I did manage to use `software-update` to install the CLT 13.2, but the App Store doesn't let you install old versions of Xcode which are somehow only available through the Developer program.

What can I do, assuming I'm not willing/able to update to Ventura?

**Comment by carlocab** [comment]
Try doing
```
sudo xcode-select --switch /Library/Developer/CommandLineTools
```
to default to your newer CLT installation.

**Reply by benknoble** [reply]
The command succeeded, but `brew upgrade` did not.

**Comment by Bo98** [comment, marked as answer]
> Xcode which are somehow only available through the Developer program.

You can download it with a free developer account. I did so yesterday.

**Reply by benknoble** [reply]
What is a "free developer account"? All I can do is Enroll ($99 USD) or go to https://developer.apple.com/download/applications/ which lists only Xcode 14.1 RC2, Xcode 14.01, and Xcode 14 (in the Xcode line).

This isn't the request version and is unlikely to work on my system given the version requirements.

**Reply by Bo98** [reply]
It's under "more" rather than "applications": https://developer.apple.com/download/all/.

Or here's a direct link: https://developer.apple.com/services-account/download?path=/Developer_Tools/Xcode_13.2.1/Xcode_13.2.1.xip
